.github/autobuild/android.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+set -eu
+
+COMMANDLINETOOLS_VERSION=6858069
+ANDROID_NDK_VERSION=r21d
+ANDROID_PLATFORM=android-30
+ANDROID_BUILD_TOOLS=30.0.2
+AQTINSTALL_VERSION=2.0.6
+QT_VERSION=5.15.2
+
+# Only variables which are really needed by sub-commands are exported.
+# Definitions have to stay in a specific order due to dependencies.
+QT_BASEDIR="/opt/Qt"
+ANDROID_BASEDIR="/opt/android"
+export ANDROID_SDK_ROOT="${ANDROID_BASEDIR}/android-sdk"
+COMMANDLINETOOLS_DIR="${ANDROID_SDK_ROOT}"/cmdline-tools/latest/
+ANDROID_NDK_ROOT="${ANDROID_BASEDIR}/android-ndk"
+ANDROID_NDK_HOST="linux-x86_64"
+ANDROID_SDKMANAGER="${COMMANDLINETOOLS_DIR}/bin/sdkmanager"
+export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64/"
+export PATH="${PATH}:${ANDROID_SDK_ROOT}/tools"
+export PATH="${PATH}:${ANDROID_SDK_ROOT}/platform-tools"
+
+if [[ ! ${jamulus_buildversionstring:-} =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    echo "Environment variable jamulus_buildversionstring has to be set to a valid version string"
+    exit 1
+fi
+
+setup_ubuntu_dependencies() {
+    export DEBIAN_FRONTEND="noninteractive"
+
+    sudo apt-get -qq update
+    sudo apt-get -qq --no-install-recommends -y install build-essential zip unzip bzip2 p7zip-full curl chrpath \
+        libxkbcommon-x11-0 openjdk-8-jre-headless openjdk-8-jdk-headless
```

Comment:
Why do we need libxkbcommon-x11? According to the Debian documentation this is related to the keyboard somehow?

Reply:
No idea. Seems like it was part of the initial addition in a0a2f1205. I can try to remove it and see what happens.
jre is also redundant if jdk is installed.